pages/conduct/reporting.md

```diff
@@ -0,0 +1,50 @@
+
+In your report please include:
+
+*   Your contact info (so we can get in touch with you if we need to follow up)
+*   Names (real, nicknames, or pseudonyms) of any individuals involved. If there were other witnesses besides you, please try to include them as well.
+*   When and where the incident occurred. Please be as specific as possible.
+*   Your account of what occurred. If there is a publicly available record (e.g. a mailing list archive or a public IRC logger) please include a link.
+*   Any extra context you believe existed for the incident.
+*   If you believe this incident is ongoing.
+*   Any other information you believe we should have.
+
+### What happens after you file a report?
+
+You will receive an email from the jQuery Foundation Code of Conduct Working Group acknowledging receipt immediately. We promise to acknowledge receipt within 24 hours (and will aim for much quicker than that).
+
+The working group will immediately meet to review the incident and determine:
+
+*   What happened.
+*   Whether this event constitutes a code of conduct violation.
+*   Who the bad actor was.
+*   Whether this is an ongoing situation, or if there is a threat to anyone's physical safety.
+
+If this is determined to be an ongoing incident or a threat to physical safety, the working groups' immediate priority will be to protect everyone involved. This means we may delay an "official" response until we believe that the situation has ended and that everyone is physically safe.
+
+Once the working group has a complete account of the events they will make a decision as to how to response. Responses may include:
+
+*   Nothing (if we determine no violation occurred).
+*   A private reprimand from the working group to the individual(s) involved.
+*   A public reprimand.
+*   An imposed vacation (i.e. asking someone to "take a week off" from a mailing list or IRC).
+*   A permanent or temporary ban from some or all jQuery spaces (mailing lists, IRC, etc.)
+*   A request for a public or private apology.
+
+We'll respond within one week to the person who filed the report with either a resolution or an explanation of why the situation is not yet resolved.
+
+Once we've determined our final action, we'll contact the original reporter to let them know what action (if any) we'll be taking. We'll take into account feedback from the reporter on the appropriateness of our response, but we don't guarantee we'll act on it.
+
+Finally, the Working Group will make a report on the situation to the jQuery Foundation board. The board may choose to a public report of the incident.
+
+### Appealing
+
+Only permanent resolutions (such as bans) may be appealed. To appeal a decision of the working group, contact the jQuery Foundation Board at [foundation@jquery.com](mailto:foundation@jquery.com) with your appeal and the jQuery Foundation board will review the case.
```

Comment:
@gnarf +1 you cant have appeal something to the people who made the decision in the first place its a conflict of interest at that point. What you could possibly say to change their mind they made the original decision.